packages/commerce/backend/src/app.ts

```diff
@@ -99,4 +107,27 @@ export class App {
       res.status(500).json({ success: false, message: 'Internal Server Error' })
     }
   }
+
+  private async fileHandler(
```

Comment:
maybe we store the images in a separate directory and serve them as static files?